### ISSUE TITLE
Unused vars, imports not detected

### ISSUE THREAD

**Issue by mosesoak** [Microsoft/vscode-tslint#219]
Possible cause:

Newest version of tslint requires that the --type-check and --project flags are specified in order to show unused imports and variables.

**Comment by Bnaya** [comment]
turns out you need to add:
```
    "noUnusedLocals": true,
    "noUnusedParameters": true,
```
in your tsconfig

**Comment by BenDTU** [comment]
That's still not working for me.

**Comment by mosesoak** [comment]
@Bnaya doesn't work for me either. I tried setting those as compiler options and rules (in kebab-case), no dice.

The basic rule https://palantir.github.io/tslint/rules/no-unused-variable/ is supposed to flag unused imports but doesn't for me.

Fails in both of these: `Version 1.13.1 (1.13.1)` or `Version 1.14.0-insider (1.14.0-insider)`

**Comment by egamma** [comment]
tslint version 5 requires  --type-check for the unuseVariable rule. This is not supported by vscode-tslint and is discussed heree https://github.com/Microsoft/vscode-tslint/issues/70#issuecomment-241041929. 

The workaround is to use tslint version 4 or to use the TypeScript compiler options  "noUnusedLocals" and  "noUnusedParameters".
